debootstrap-ng.sh

```diff
@@ -288,7 +288,7 @@ prepare_partitions()
 	parttype[btrfs]=btrfs
 	# parttype[nfs] is empty
 
-	mkopts[ext4]='-q -m 2'
+	mkopts[ext4]='-O ^64bit,^metadata_csum,uninit_bg -q -m 2'
```

Comment:
`NO_APT_CACHER=yes` parameter handling should be fixed, but this doesn't change the fact that these ext4 options break process on Trusty (which is still declared as a compatible build host) and it's too early to drop Trusty support since AFAIK Trusty->Xenial upgrade will be available after 16.04.1 release (scheduled end of July).